src/alltoall.cu

```diff
@@ -68,12 +68,13 @@ testResult_t AlltoAllRunColl(void* sendbuff, void* recvbuff, size_t count, ncclD
   printf("NCCL 2.7 or later is needed for alltoall. This test was compiled with %d.%d.\n", NCCL_MAJOR, NCCL_MINOR);
   return testNcclError;
 #else
-  NCCLCHECK(ncclGroupStart());
-  for (int r=0; r<nRanks; r++) {
-    NCCLCHECK(ncclSend(((char*)sendbuff)+r*rankOffset, count, type, r, comm, stream));
-    NCCLCHECK(ncclRecv(((char*)recvbuff)+r*rankOffset, count, type, r, comm, stream));
-  }
-  NCCLCHECK(ncclGroupEnd());
+  NCCLCHECK(ncclAllToAll(sendbuff, recvbuff, count, type, comm, stream));
+  // NCCLCHECK(ncclGroupStart());
+  // for (int r=0; r<nRanks; r++) {
+  //   NCCLCHECK(ncclSend(((char*)sendbuff)+r*rankOffset, count, type, r, comm, stream));
+  //   NCCLCHECK(ncclRecv(((char*)recvbuff)+r*rankOffset, count, type, r, comm, stream));
+  // }
+  // NCCLCHECK(ncclGroupEnd());
```

Comment:
Remove the comments

Reply:
I assume those comments are moving into the baseline implementation of all2all in NCCL itself.

Reply:
correct!

Reply:
So, I will keep them as is so that we can switch back and forth. @madanMus, I will eventually make two explicit implementations for SCKL version and NCCL version to make the comparison easier, but for now, this is fine.